spring-integration-file/src/main/java/org/springframework/integration/file/config/RemoteFileOutboundChannelAdapterParser.java

```diff
@@ -48,9 +48,18 @@ protected AbstractBeanDefinition parseConsumer(Element element, ParserContext pa
 		if (StringUtils.hasText(mode)) {
 			handlerBuilder.addConstructorArgValue(mode);
 		}
+		postProcessBuilder(handlerBuilder, element);
 		return handlerBuilder.getBeanDefinition();
 	}
 
+	protected Class<?> handlerClass() {
+		return FileTransferringMessageHandler.class;
+	}
+
+	protected void postProcessBuilder(BeanDefinitionBuilder builder, Element element) {
+		// no-op
```

Comment:
Nit-pick, but another similar method in the `AbstractRemoteFileOutboundGatewayParser` is without this comment.
Or everywhere, or nowhere.